fitbenchmarking/fitbenchmarking
# Adding fitbenchmarking.com css style

#### Description of Work

Fixes #414 
Fixes #384 

Pulls out style sheet from `fitbenchmarking.com` and adds back buttons to output.

#### Testing Instructions

1. Check the style of the results are the same as `fitbenchmarking.com`.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
